launcher/README.md

```diff
@@ -1,24 +1,40 @@
-The preflight updater GUI currently supports both PyQt4 and PyQt5. To
-enforce the use of PyQt5, set the environment variable SDW_UPDATER_QT to 5.
+# securedrop-workstation updater
 
-## Why support PyQt4 and PyQt5?
+The Updater ensures that the SecureDrop Workstation is up-to-date by checking for and applying any necessary VM updates, which may prompt a reboot.
 
-Qubes 4.0.3 uses an end-of-life Fedora template in dom0 (fedora-25). See
-rationale here:
+## Running the Updater
 
-https://www.qubes-os.org/doc/supported-versions/#note-on-dom0-and-eol
+Qubes 4.0.3 uses an end-of-life Fedora template in dom0 (fedora-25). See rationale here: https://www.qubes-os.org/doc/supported-releases/#note-on-dom0-and-eol.
 
-fedora-25 only includes PyQt4, which is why we have to support it for now.
-The next version of Qubes, Qubes 4.1, will include PyQt5 in dom0.
+To run the preflight updater:
```

Comment:
It may be useful to explicitly say "on a Debian Buster system" since those instructions won't work e.g. on Ubuntu 20.04 (no more PyQt4 packages :/).

Reply:
Since the step below says to open a `dom0` terminal, I could say instead, open a `Qubes 4.0.4 dom0` terminal. That should cover it.

Reply:
Sorry, I commented on the wrong line - I meant to respond to the instructions "To run the preflighter outside dom0" later in the doc, specifically re: PyQt4 - in that context I think it might be useful to specify that a Debian Buster system is required.

Reply:
Actually, we already mention how Qubes 4.0.3 (updating it now to 4.0.4) and that it uses a fedora-25 template, so i think we're good if i add back the bit about pyqt4 being the latest version of pyqt that we can install on fedora-25.